Week3/homework/step3.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+  const number = base;
+  return x => x + number;
```

Comment:
Excellent 💯 
But for shortness, you don't have to declare line number 4 but you only have to:
```javascript
function createBase(base) {
  return x => x + base;
}
```